### PR TITLE
feat: UIPlayground integration (epic/playground->redesign)

### PR DESCRIPTION
**Pull request by yadamskaya**

### CODE REVIEW

**Review by ala-n — APPROVED** [review]

File: pages/views/draft/ui-playground/media.njk [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: content
+title: ESL Media (Playground)
+seoTitle: ESL Media - custom element to embed video and audio with a different sources like MP4, YouTube, Brightcove, etc.
```
Comment (NastaLeo) [inline pages/views/draft/ui-playground/media.njk:4]:
```suggestion
seoTitle: ESL Media - custom element to embedded video and audio with a different sources like MP4, YouTube, Brightcove, etc.
```

File: pages/views/components/esl-media.njk
Comment (NastaLeo) [inline pages/views/components/esl-media.njk:1]: We don't have yet esl-d-media-source.
Can we delete related example (line 25)?

**Review by NastaLeo — APPROVED** [review]

**Comment by ala-n** [comment]
refactor(site): `@exadel/ui-playground` project integrated to site stack

Draft created for accordion, media, and scrollbar

**Pull request merged**